src/ml/neural_net/tf_compute_context.hpp

```diff
@@ -23,9 +23,7 @@ class tf_compute_context: public compute_context {
 
 
   /**
-   * Constructs a context wrapping the best currently available Metal device.
-   *
-   * \todo Guard against eGPU coming and going?
+   * Constructs a context wrapping devices that will not have Metal.
```

Comment:
I don't think there's any reason to mention Metal at all here, it just confuses things.